include/rocksdb/comparator.h

```diff
@@ -150,4 +150,6 @@ extern const Comparator* BytewiseComparator();
 // ordering.
 extern const Comparator* ReverseBytewiseComparator();
 
+extern const Comparator* ComparatorWithU64Ts();
```

Comment:
We should put the comments about its limitation here, and explicitly state it's experimental and subject to change/removal

Reply:
Also we should think more about the API name and contract.

Reply:
As commented earlier, we can avoid public API change